kasa/device.py

```diff
@@ -411,6 +389,14 @@ def _get_replacing_attr(self, module_name: ModuleName, *attrs):
         # light preset attributes
         "presets": (Module.LightPreset, ["_deprecated_presets", "preset_states_list"]),
         "save_preset": (Module.LightPreset, ["_deprecated_save_preset"]),
+        # Emeter attribues
+        "get_emeter_realtime": (Module.Energy, ["get_status"]),
+        "emeter_realtime": (Module.Energy, ["status"]),
+        "emeter_today": (Module.Energy, ["consumption_today"]),
+        "emeter_this_month": (Module.Energy, ["consumption_this_month"]),
+        "current_consumption": (Module.Energy, ["current_consumption"]),
+        "get_emeter_daily": (Module.Energy, ["get_daystat"]),
+        "get_emeter_monthly": (Module.Energy, ["get_monthstat"]),
```

Comment:
I'm wondering if we should still keep the old function naming in this case? Or have somethingh liek `get_daily_stats`/`get_monthly_stats`?